hogben/utils.py

```diff
@@ -200,8 +200,10 @@ def fisher(qs, xi, counts, models, step=0.005):
             lb = np.array([param.bounds.limits[0] for param in xi])
             ub = np.array([param.bounds.limits[1] for param in xi])
 
+        if importance is None: # Default importance equals one for each param
+            importance = np.ones(len(lb))
         # Using the equations from the paper for the coordinate transform.
-        H = np.diag(1/(ub-lb))
+        H = np.diag(importance/(ub-lb))
         g = np.dot(np.dot(H.T, g), H)
 
```

Comment:
Please add a unit test for this

Reply:
I added a unit test for this, but this does require the `test_fisher` module to exist. I've added the basic requirements for that, and test the case with and without this scaling. I could move that to PR #55 if prefered